.babelrc

```diff
@@ -3,8 +3,5 @@
     "es2015",
     "react",
     "stage-1"
-  ],
-  "plugins": [
-    "add-module-exports"
```

Comment:
Removed this plugin in favor of using vanilla ES6 modules and `require().default` where necessary.